fix(admin permissions)

mathemagica opened the pull request:
if a user doesnt have access to this admin page,  redirect to the homepage

DBNess commented:
Just tested this locally :+1: and pushed it to `girldevelopit:redirect_non_admins`
Improves the error feedback signficantly - thanks @mathemagica!

![girl_develop_it](https://cloud.githubusercontent.com/assets/226228/5762570/e39e49b0-9cb0-11e4-9fb4-386daab9d817.jpg)

mathemagica commented:
Just added changes to update db/seeds.rb to ensure the admin user 'julia@girldevelopit.com' has the admin role added. Also added a migration that just in times adds the admin role to 'julia@girldevelopit.com' if this user doesnt already have this role.

DBNess commented:
This is fixed! Shipped #102 (contributed by @mathemagica) and added these permissions to Julia's account on production. Thanks @mathemagica :clap: :tada:

therufs commented:
\o/ ​
